Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:centos7
+MAINTAINER Stech Training
+
+RUN yum -y update \
+    #Install Python repo, Python 3.6  and some addons
+    && yum -y install https://centos7.iuscommunity.org/ius-release.rpm \
+    && yum -y install python36u \
+    && yum -y install python36u-libs \
+    && yum -y install python36u-devel \
+    && yum -y install python36u-pip \

```

Comment:
you can put these all in one line, e.g.
yum install -y python36u python36u-libs python36u-devel install python36u-pip \